Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml

```diff
@@ -249,7 +252,12 @@ script:
                     };
                 }
             }
-            var fullResult = [{Type: entryTypes.note, Contents: jsonRes, ContentsFormat: formats.json, HumanReadable: md, EntryContext: context}];
+            if (verboseEnrichment === false) {
+                var fullResult = [{Type: entryTypes.note, Contents: context, ContentsFormat: formats.json, HumanReadable: md, EntryContext: context}];
```

Comment:
The `context` here contains stuff like DBotScore and Domain. I don't see why we should return it in `Contents`
What exactly is returned in non verboseEnrichment?

Reply:
Changed the logic to show 20 keys in the md and in the raw-response.